### ISSUE TITLE
Titles on Links should be warning, not error

### ISSUE THREAD

**Issue by Elpie** [redroot/holmes#8]
Titles in links are not a requirement of WCAG 2.0. Providing link text that describes the purpose of the link is a requirement and that _may_ require titles in some limited cases. The lack of titles should be a warning so users look more carefully at their links to check if they can be better written or if titles are needed to provide context. 

Leaving titles out is never an error. Not writing good link text is an error but this is something that can't be tested since the tests cannot understand context. 

For more information see: http://www.w3.org/TR/WCAG20-TECHS/H33.html
